govcd/vm.go

```diff
@@ -175,7 +175,18 @@ func (vm *VM) PowerOff() (Task, error) {
 
 }
 
-func (vm *VM) ChangeCPUcount(size int) (Task, error) {
+// Sets number of available virtual logical processors
+// (i.e. CPUs x cores per socket)
```

Comment:
Probably worth mentioning that number of cores is inherited from template, using this function.

Reply:
added